fastlane/env/user.env-example

```diff
@@ -1,3 +1,7 @@
 GHHELPER_ACCESS=
 APPCENTER_API_TOKEN=
-SENTRY_AUTH_TOKEN=
\ No newline at end of file
+SENTRY_AUTH_TOKEN=
+
+# These should point to the same path
+APP_STORE_CONNECT_API_KEY_PATH=
+SIGH_API_KEY_PATH=
```

Comment:
If I'll have time this week or the next, I'll try opening a PR on Fastlane to make `sigh` (used by `match` under the hood) fallback to `APP_STORE_CONNECT_API_KEY_PATH` if `SIGH_API_KEY_PATH` is not given.

Reply:
Oh this is a great suggestion, thanks! I started with `user.env` to get it to work locally and forgot about the `project.env` option. 🙇‍♂️

Reply:
I started implementing this when I thought... why do we need to set it as an environment variable in the first place? I mean the file itself is encrypted, and `deliver` has an `api_key_path` parameter.

Maybe it would be simpler and more direct to pass the path using that parameter on the `deliver` call site. What do you think?

I might be missing some rationale on why we're using the environment variable. Summoning @jkmassel for that.

Thanks.

Reply:
I thought about it too and the only rationale I could think of is so that it can work and be overridden by people not using the mobile secrets, like external contributors. But then I can't see why non A12s would want to or be allowed to run those lanes in the first place, so yeah…

Reply:
True 🤔 

I don't think there's a good ROI in keeping the release process flexible so that an external contributor can run it.

Let's see what @jkmassel has do say about this...

Reply:
> I don't think there's a good ROI in keeping the release process flexible so that an external contributor can run it.

I agree with this – using the file path in `.configure-files` is probably best.